rosdep/python.yaml

```diff
@@ -3574,6 +3577,9 @@ python-sympy:
     wily_python3: [python3-sympy]
     xenial: [python-sympy]
     xenial_python3: [python3-sympy]
+python-sysv-ipc:
+  ubuntu:
+    xenial: [python-sysv-ipc]
```

Comment:
This is available on jessie and newer and should have a rule: https://packages.debian.org/jessie/python-sysv-ipc

Also it's on xenial and newer too: https://packages.ubuntu.com/xenial/python-sysv-ipc

Fedora and gentoo rules would be appreciated as well.